portfolio/src/main/webapp/script.js

```diff
@@ -45,10 +45,12 @@ function displayTab(event, tabName) {
  */
 function drawTimeline() {
     hideImages();
+    hideBranches();
     var timeline = document.getElementById("timeline");
     var movingDot = document.getElementById("moving-dot")
     var rightEnd = 95;
     var dotLeftPos = 97 - rightEnd;
+    var branchPosToExtend = 0;
```

Comment:
Please add comments explaining complex steps, for example, what you are achieving by extendTimeline is unclear. Alternatively you can choose a better name, by subtracting rightend by 0.5, I think you are moving through the timeline.